Library/Formula/svtplay-dl.rb

```diff
@@ -22,6 +15,7 @@ class SvtplayDl < Formula
   end
 
   depends_on "rtmpdump"
+  depends_on "python"
```

Comment:
10.9 and 10.10.4 ..  looking at http://www.opensource.apple.com  i can see that in 10.10.5 have python 2.7.10